subsys/net/lib/sockets/sockets_select.c

```diff
@@ -197,3 +200,75 @@ int zsock_select(int nfds, zsock_fd_set *readfds, zsock_fd_set *writefds,
 
 	return num_selects;
 }
+
+#ifdef CONFIG_USERSPACE
+static int z_vrfy_zsock_select(int nfds, zsock_fd_set *readfds,
+			       zsock_fd_set *writefds,
+			       zsock_fd_set *exceptfds,
+			       struct zsock_timeval *timeout)
```

Comment:
Shouldn't this function copy the `timeout` parameter as well?

Reply:
> Shouldn't this function copy the `timeout` parameter as well?

Yep, it should, will fix.